parity/cli/mod.rs

```diff
@@ -868,6 +868,18 @@ usage! {
 			"--pruning-memory=[MB]",
 			"The ideal amount of memory in megabytes to use to store recent states. As many states as possible will be kept within this limit, and at least --pruning-history states will always be kept.",
 
+			ARG arg_storage_writing: (String) = "off", or |c: &Config| c.footprint.as_ref()?.storage_writing.clone(),
+			"--storage-writing=[DATABASE]",
+			"Enable writing storage trie diffs to secondary database. DATABASE may be one of csv or off.",
+
+			ARG arg_storage_writing_path: (Option<String>) = None, or |c: &Config| c.footprint.as_ref()?.storage_writing_path.clone(),
+			"--storage-writing-path=[PATH]",
+			"Specify the storage diff writing directory path",
+
+			ARG arg_watched_contracts: (Option<String>) = None, or |c: &Config| c.account.as_ref()?.unlock.as_ref().map(|vec| vec.join(",")),
+			"--watch-contracts=[ACCOUNTS]",
```

Comment:
if this flag isnt set does it just watch all accounts?

Reply:
yea i think it would be nice to give the user the ability to easily watch all contracts, otherwise they would have to list all the contracts lol. i didnt think about the consequences of watching all the contracts though, i think pointing that out like you said would be nice.